src/containers/Header.tsx

```diff
@@ -0,0 +1,5 @@
+import { connect } from "react-redux";
+import { actions } from "../appSlice";
+import Header from "../components/Header";
+
+export default connect(null, { addTodo: actions.addTodo })(Header);
```

Comment:
I am not sure if using `connect` is still the way to go currently, but it seems like you are still able to use it. (also applies to `mapStateToProps` and `mapDispatchToProps`)